tt_llk_blackhole/common/inc/sfpu/ckernel_sfpu_where.h

```diff
@@ -0,0 +1,100 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "sfpi.h"
+
+namespace ckernel
+{
+namespace sfpu
```

Comment:
```suggestion
namespace ckernel::sfpu
```
We tend to use new goodies from C++17 now.